server-foundation/foundation-sop.md

```diff
@@ -1,15 +1,24 @@
 # ACM Foundation SOPs for SD
 
-## ManagedCluster
-- [Cluster importing is not finished in 10 minutes](./alerts/manaegdcluster.md#cluster-importing-is-not-finished-in-10-minutes)
-- [Managed cluster keeps unreachable for 10 minutes](./alerts/manaegdcluster.md#managed-cluster-keeps-unreachable-for-10-minutes)
-- [Kube-apiserver of managed cluster keeps unavailable for 10 minutes](./alerts/manaegdcluster.md#kube-apiserver-of-managed-cluster-keeps-unavailable-for-10-minutes)
-- [Rotation of managed cluster client cert keeps failed in 24 hours](./alerts/manaegdcluster.md#rotation-of-managed-cluster-client-cert-keeps-failed-in-24-hours)
+There are several critical informations you need to get before you start troubleshooting, and they can be found in alerts' annotation -- `description`.
+* managedcluster_name
+* manifestwork_name
+
+The you need to use input the `managed_cluster_name` as the dashboard variable `managedcluster` and check the `ManagedCluster Finder` row.
```

Comment:
What's the name of the dashboard mentioned here, and how to access it (like url)?